tests/pkg_ubasic/Makefile

```diff
@@ -0,0 +1,15 @@
+include ../Makefile.tests_common
+
+TEST_ON_CI_WHITELIST += all
+
+USEPKG += ubasic
+
+# ubasic_init_peek_poke used in those tests is not exposed
+CFLAGS += -Wno-implicit-function-declaration
+
+include $(RIOTBASE)/Makefile.include
+
+clean: ..rm-main
```

Comment:
what's this?

Reply:
Remainders of my try not having include the foreign tests.c...

Reply:
(I removed that part in a fixup)

Reply:
(and readded it after https://github.com/RIOT-OS/RIOT/pull/11319#discussion_r271506808)

Reply:
I think I'd prefer an empty main.c, or as empty as gcc permits.

Fiddling with files within the source tree is usually a bad idea. E.g., CI builds multiple boards in the same checkout.

I suggest including tests.c in the pkg makefile depending on a make variable. "main()" doesn't have to be in the application folder.

Reply:
Well, @cladmi had also the idea of introducing a pseudo-module (e.g. `ubasic_tests`) for that, as it is the more RIOT-y way to go with that (and wouldn't introduce YAER (Yet Another Exception from the Rule). I did not like that idea, as it would be to messy (aka require my to modify `pseudomodules.mk` as well). If we can live with that, I can do it that way.

Reply:
Found a way to do it without any pseudomodules or file fiddling.